doc/user-guide/push-notifications/Push-notifications-client-side.md

```diff
@@ -112,25 +112,33 @@ To enable push notifications in the simplest configuration, just send the follow
 We have now enabled push notifications to be send to the `pubsub.mypubsub.com` domain
 on the node `punsub_node_for_my_private_iphone` created previously, or in the case of PubSub-less,
 for whatever unique node name we give here, for example any variation of the token obtained from
-_APNS_ or _FCM_. In `publish-options` we have passed the service name we are using (`apns` or `fcm`)
-and the device token (here: `your_pns_device_token`) that you received from your push notification
-service provider (as described in [Registering with Push Service provider](#registering-with-a-push-service-provider)).
-Those two options are the only ones required, but there are some others that are optional:
+_APNS_ or _FCM_. Please note that `publish-options` are specific to various XMPP Push Services.
+
+### Publish options
+For [mod_push_service_mongoosepush][] the next `publish-options` are mandatory:
+
+  * `device_id` - device token (here: `your_pns_device_token`) that you received from your push notification service provider (as described in [Registering with Push Service provider](#registering-with-a-push-service-provider))
+  * `service` - push notification service provider name (`apns` or `fcm`)
+
+there are also some other `publish-options` supported:
```

Comment:
I think, it'd be good to mention that other options are simply ignored (by current `mod_push_service_mongoosepush`) and will never make it to the MongoosePush.